### ISSUE TITLE
Access to /system/indices throws an error

### ISSUE THREAD

**Issue by bernd** [Graylog2/graylog2-server#1373]
After an update from 1.1.6 to 1.2.0-rc.1 I get the following error when trying to access the `/system/indices` URL to recalculate the index ranges.

![gczsdepgybaqcagebaicaqeagkbl4oaipffzqhfrwucaggbgebaicaqeagibd4laolef5arfp0qcagebaicaygaqeagibd4mgj8p5y8us78h6guaaaaaelftksuqmcc](https://cloud.githubusercontent.com/assets/461/9491525/14de5756-4bf0-11e5-8053-bdd29fef0f8d.png)
```
org.graylog2.restclient.lib.ApiClientImpl$ApiRequestBuilder#execute (ApiClientImpl.java:457)
org.graylog2.restclient.models.IndexService#all (IndexService.java:50)
controllers.IndicesController#index (IndicesController.java:74)
Routes$$anonfun$routes$1$$anonfun$applyOrElse$60$$anonfun$apply$614#apply (routes_routing.scala:2069)
Routes$$anonfun$routes$1$$anonfun$applyOrElse$60$$anonfun$apply$614#apply (routes_routing.scala:2069)
play.core.Router$HandlerInvokerFactory$$anon$4#resultCall (Router.scala:264)
play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1#invocation (Router.scala:255)
play.core.j.JavaAction$$anon$1#call (JavaAction.scala:55)
play.GlobalSettings$1#call (GlobalSettings.java:67)
play.mvc.Security$AuthenticatedAction#call (Security.java:44)
play.core.j.JavaAction$$anonfun$11#apply (JavaAction.scala:82)
play.core.j.JavaAction$$anonfun$11#apply (JavaAction.scala:82)
scala.concurrent.impl.Future$PromiseCompletingRunnable#liftedTree1$1 (Future.scala:24)
scala.concurrent.impl.Future$PromiseCompletingRunnable#run (Future.scala:24)
play.core.j.HttpExecutionContext$$anon$2#run (HttpExecutionContext.scala:40)
play.api.libs.iteratee.Execution$trampoline$#execute (Execution.scala:46)
play.core.j.HttpExecutionContext#execute (HttpExecutionContext.scala:32)
scala.concurrent.impl.Future$#apply (Future.scala:31)
scala.concurrent.Future$#apply (Future.scala:485)
play.core.j.JavaAction$class#apply (JavaAction.scala:82)
play.core.Router$HandlerInvokerFactory$JavaActionInvokerFactory$$anon$15$$anon$1#apply (Router.scala:252)
play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5#apply (Action.scala:130)
play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4$$anonfun$apply$5#apply (Action.scala:130)
play.utils.Threads$#withContextClassLoader (Threads.scala:21)
play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4#apply (Action.scala:129)
play.api.mvc.Action$$anonfun$apply$1$$anonfun$apply$4#apply (Action.scala:128)
scala.Option#map (Option.scala:145)
play.api.mvc.Action$$anonfun$apply$1#apply (Action.scala:128)
play.api.mvc.Action$$anonfun$apply$1#apply (Action.scala:121)
play.api.libs.iteratee.Iteratee$$anonfun$mapM$1#apply (Iteratee.scala:483)
play.api.libs.iteratee.Iteratee$$anonfun$mapM$1#apply (Iteratee.scala:483)
play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1#apply (Iteratee.scala:519)
play.api.libs.iteratee.Iteratee$$anonfun$flatMapM$1#apply (Iteratee.scala:519)
play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14#apply (Iteratee.scala:496)
play.api.libs.iteratee.Iteratee$$anonfun$flatMap$1$$anonfun$apply$14#apply (Iteratee.scala:496)
scala.concurrent.impl.Future$PromiseCompletingRunnable#liftedTree1$1 (Future.scala:24)
scala.concurrent.impl.Future$PromiseCompletingRunnable#run (Future.scala:24)
akka.dispatch.TaskInvocation#run (AbstractDispatcher.scala:41)
akka.dispatch.ForkJoinExecutorConfigurator$AkkaForkJoinTask#exec (AbstractDispatcher.scala:393)
scala.concurrent.forkjoin.ForkJoinTask#doExec (ForkJoinTask.java:260)
scala.concurrent.forkjoin.ForkJoinPool$WorkQueue#runTask (ForkJoinPool.java:1339)
scala.concurrent.forkjoin.ForkJoinPool#runWorker (ForkJoinPool.java:1979)
scala.concurrent.forkjoin.ForkJoinWorkerThread#run (ForkJoinWorkerThread.java:107)
```

**Comment by bernd** [comment]
Maybe it's related to `` being slow?

```
$ time curl -n localhost:12900/system/indices/ranges                                                                                  
[...]
real    0m29.785s
user    0m0.007s
sys     0m0.000s
`

**Comment by bernd** [comment]
The issue seems to be the search request to receive the index ranges.

```java
       final SearchRequest request = client.prepareSearch()
                .setTypes(IndexMapping.TYPE_INDEX_RANGE)
                .setIndices(indices.allIndicesAlias())
                .setQuery(QueryBuilders.matchAllQuery())
                .setSize(Integer.MAX_VALUE)
                .request();
```
See: https://github.com/Graylog2/graylog2-server/blob/master/graylog2-server/src/main/java/org/graylog2/indexer/ranges/EsIndexRangeService.java#L151-L156

If I didn't get it wrong, this is the equivalent curl request:

```
time curl "localhost:9200/graylog2_*/index_range/_search?size=2147483647
[...]
real	0m22.852s
user	0m0.006s
sys	0m0.004s
```

It seems faster without the huge `size` parameter:

```
$ time curl -s "localhost:9200/graylog2_*/index_range/_search?pretty=true&size=2147483647" >/dev/null

real	0m30.935s
user	0m0.004s
sys	0m0.004s
$ time curl -s "localhost:9200/graylog2_*/index_range/_search?pretty=true&size=100" >/dev/null

real	0m6.381s
user	0m0.006s
sys	0m0.002s
```

**Comment by bernd** [comment]
This is with Elasticsearch 1.6.0.

**Comment by bernd** [comment]
Fixed by #1374
